modules/ROOT/pages/advcode.adoc

```diff
@@ -34,6 +39,21 @@ tinymce.init({
 });
 ----
 
+== Example: enable the {pluginname} full-screen mode button
+
+[source,js]
+----
+tinymce.init({
+  selector: "textarea",  // change this value according to your HTML
+  advcode_inline: true,
+  plugins: [
+    "code", "fullscreen", 'advcode',
```

Comment:
```suggestion
    "fullscreen", 'advcode',
```
The `code` plugin cannot be used with the `advcode` plugin as they both use the same button.